src/Common/ProcessExecutor.php

```diff
@@ -15,11 +15,27 @@ class ProcessExecutor implements ConfigAwareInterface, LoggerAwareInterface, Out
     use ProgressIndicatorAwareTrait;
     use OutputAwareTrait;
 
+    /**
+     * @param Process $process
+     * @return type
+     */
     public function __construct(Process $process)
     {
         $this->process = $process;
     }
 
+    public static function create($container)
+    {
+        $processExecutor = new self($commandFile);
```

Comment:
Typo -- missing an argument and etc.  I guess there are no tests for ProcessExecutor?